### PR TITLE
Feature/vaccine index screen

### PR DESCRIPTION
**Pull request by ErikAlluxi**
- Vaccine Index screen creation 
- Card vaccine componet creation
- I added 2 new Icons to use them
- I added the Vaccine index screen to home.navigator component

### CODE REVIEW

File: src/myapp/layouts/vaccines/vaccine-index/index.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+//UI Kitten
+import {Layout, Text, StyleService, List} from '@ui-kitten/components';
+
+export default ({navigation, route}): React.ReactElement => {
+  const {id, breed, name, pet_age, sex} = route.params.pet;
+
+  /* const vaccines = null;
+  const vaccines2 = null; */
+
+  const vaccines = [
```
Comment (gusreyes01) [inline src/myapp/layouts/vaccines/vaccine-index/index.tsx:19]: @ErikAlluxi this is just test data, correct? the real one will be coming from the API

**Review by gusreyes01 — COMMENTED** [review]
Please review my comment

File: src/myapp/layouts/vaccines/vaccine-index/index.tsx (hunk 1 above)
Comment (ErikAlluxi) [reply src/myapp/layouts/vaccines/vaccine-index/index.tsx:19]: @gusreyes01 Yes, that is test data, I´m going to replace with the API call, I just leave it to show info in the cards, and cliens can see it

**Comment by gusreyes01** [comment]
Thanks @ErikAlluxi it looks good but there's some conflicts. Please correct them and I will merge the PR.

**Comment by ErikAlluxi** [comment]
@gusreyes01 I´ve already relsoved the conflicts, could you please review them again?

**Pull request merged**